ethexe/processor/src/host/api/mod.rs

```diff
@@ -58,7 +59,7 @@ impl MemoryWrap {
         store: impl Into<StoreContext<'a, T>>,
         ptr_len: i64,
     ) -> &'a [u8] {
-        let [ptr, len]: [i32; 2] = unsafe { mem::transmute(ptr_len) };
+        let (ptr, len) = unpack_i64(ptr_len);
```

Comment:
Not sure if this is intended, but it seems `unpack_i64` and `mem::transmute::<i64, [i32; 2]>` do not produce the same byte transformation on little-endian architectures.